package.json

```diff
@@ -5,6 +5,9 @@
   "version": "1.5.1",
   "license": "Apache-2.0",
   "author": "Google Inc.",
+  "engines": {
+    "node": ">=6.2.2"
```

Comment:
So right now, the only requirement for our library is Node 6.
We have tested it to work with Node.js 6.0.0

Can you change it to "node": ">=6.0.0"? Is there a specific reason you put 6.2.2?